docs/source/release/v6.3.0/index.rst

```diff
@@ -1,22 +1,77 @@
 .. _v6.3.0:
 
+.. include:: <isonum.txt>
+
 ===========================
 Mantid 6.3.0 Release Notes
 ===========================
 
-.. figure:: ../../images/ImageNotFound.png
+.. figure:: ../../images/engggui_texture.png
    :class: screenshot
-   :width: 385px
+   :width: 350px
    :align: right
 
 .. contents:: Table of Contents
    :local:
 
-.. warning:: This release is still under construction. The changes can be found in the nightly builds on the `download page`_.
+Summary
+-------
 
 We are proud to announce version 6.3.0 of Mantid.
 
-**TODO: Add paragraph summarizing big changes**
+Much of this version provides improvements for users, building on exisiting algorithms and GUIs.
+Plotting has also seen a number of updates and users will benefit from improved documentation too.
+
+In addition to many improvements we are delighted to announce some new features including:
+
+- Full cross-platform support for Conda packages. See Conda_ for further details.
+- The :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>` now supports two texture groupings, ``Texture20`` and ``Texture30`` (see image above).
+- The Error Reporter can now remember and prefill the user's name and e-mail address.
+- The :ref:`InstrumentViewer` :ref:`Pick Tab<instrumentviewer_pick_tab>` has a new panel for allowing the rebinning of workspaces.
+- In the :ref:`Frequency Domain Analysis <Frequency_Domain_Analysis-ref>` and the :ref:`Muon Analysis <Muon_Analysis-ref>` GUIs the confidence interval of a fit, previously shown as an error bar, is now represented by a shaded region.
+- Several new fitting functions for Muons have been provided.
+
+Conda
+-----
+
+.. figure:: ../../images/conda_logo.svg
+   :align: right
```

Comment:
Can we do this above or below, it squashes the paragraph and forces me to read in short snippets:
![image](https://user-images.githubusercontent.com/1817032/152546059-ec4f6d9e-f6c3-44db-985b-1af356460f98.png)